spec/unit/installer/xcode/target_validator_spec.rb

```diff
@@ -273,17 +273,16 @@ def create_validator(sandbox, podfile, lockfile, integrate_targets = false)
               use_frameworks!
               pod 'OrangeFramework', :path => (fixture_path + 'orange-framework').to_s
               pod 'matryoshka',      :path => (fixture_path + 'matryoshka').to_s
-              target 'SampleProject'
```

Comment:
updated most of these tests *not* to use stabbing. Much better imo. Added tests for the new logic here too.

Reply:
much much cleaner IMO